presto-docs/src/main/sphinx/functions.rst

```diff
@@ -10,6 +10,9 @@ Ensure to also consult our reference to the :doc:`supported SQL data
 types</language>` and other general characteristics and a :doc:`full SQL
 statement and syntax reference</sql>` are available.
 
+If you are looking for more information about a specific function or operator,
+:doc:`take advantage of our complete list</functions/list>`.
```

Comment:
We don't write "advertising" language like this in the documentation. Replace "take advantage of our" with "see the complete ..."

Reply:
See the complete ... is typical but grammatically awful. Can we use something better and less dry to read?

Reply:
What’s wrong with “see the [complete list of functions]” or “see the [function index]”?

With brackets being the link text. This fits the existing style of the documentation.

Reply:
I updated that now. I just think it is really dry and boring so i try not to use it ..